/*
 * Copyright (c) 2023-present, FriendliAI Inc. All rights reserved.
 */

syntax = "proto3";

message V1TextToImageRequest {
    string prompt = 1;  
    optional string negative_prompt = 2;
    optional int32 num_outputs = 3;
    optional int32 num_inference_steps = 4;
    optional float guidance_scale = 5;
    optional int32 seed = 6;
    optional string response_format = 7;
}
